templates/web/base/admin/templates.html

```diff
@@ -0,0 +1,26 @@
+[% INCLUDE 'admin/header.html' title=tprintf(loc('Response Templates for %s'), body.name) -%]
```

Comment:
Ah, perhaps just an IF in the base template, then? Given it's otherwise identical.